src/main/resources/application.properties

```diff
@@ -1,7 +1,8 @@
+
 spring.jpa.hibernate.ddl-auto=update
-spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3306/Employee
+spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3306/db_employee
```

Comment:
This is a minor comment, but I'd prefer having the db name as `Employee`.
Because it is obvious that it is a database.

This is the equivalent argument of not naming your fields with the type in the prefix or suffix, like nameStr etc.

Reply:
Yes will change it back.